app/scripts/hglib.js

```diff
@@ -148,8 +147,54 @@ export const trackViewer = (element, [xMin, xMax, yMin, yMax], trackConfig) => {
       },
     ],
   };
-  return {
-    id,
-    hgApi: viewer(element, viewConfig, { bounded: true })
-  };
+  const hgApi = viewer(element, viewConfig, { bounded: true });
+  console.warn('>>> hgApi in trackViewer?', hgApi);
+  return { id, hgApi };
 };
+
+/**
+ * React component wrapper around trackViewer.
+ * Accepts x, y, width, and height props, in addition to trackConfig,
+ * so HiGlass can be used to provide background imagery for Deck.gl.
+ */
+export class HiGlassTrackComponent extends React.Component {
```

Comment:
This is should maybe be in its own file?

Reply:
done.